src/target/riscv/riscv-013.c

```diff
@@ -2570,6 +2570,18 @@ static int execute_fence(struct target *target)
 	return ERROR_OK;
 }
 
+static void log_memory_access128(target_addr_t address, uint64_t value_h,
+		uint64_t value_l, bool read)
+{
+	if (debug_level < LOG_LVL_DEBUG)
+		return;
+
+	char fmt[80];
+	sprintf(fmt, "M[0x%" TARGET_PRIxADDR "] %ss 0x%%016" PRIx64 "%%016" PRIx64,
+			address, read ? "read" : "write");
+	LOG_DEBUG(fmt, value_h, value_l);
+}
+
 static void log_memory_access(target_addr_t address, uint64_t value,
 		unsigned size_bytes, bool read)
```

Comment:
(same as above)

```suggestion
static void log_memory_access(target_addr_t address, uint64_t value,
		unsigned size_bytes, bool is_read)
```

Reply:
There are other places in this file that also use bool read, should I change all of them to is_read in this patch